code/modules/mob/living/carbon/human/species_types/plasmamen.dm

```diff
@@ -69,7 +69,9 @@
 
 	if(!preference_source)
 		return
-	var/path = J.species_outfits[SPECIES_PLASMAMAN]
+	var/path = J.species_outfits?[SPECIES_PLASMAMAN]
+	if (!path) //Somehow we were given a job without a plasmaman suit, use the default one so we don't go in naked!
+		path = /datum/outfit/plasmaman
```

Comment:
You should log an error here so that we know a job path is missing a plasmaman species. The last thing we want to do it hide actual errors.

Reply:
Ah- had a feeling I was missing something like that.
Should I log it under the job system or game system?

Reply:
Use stack_trace so it goes into runtime log